src/main/java/net/atos/entng/calendar/helpers/EventHelper.java

```diff
@@ -293,7 +293,9 @@ public void handle(final UserInfos user) {
                     @Override
                     public void handle(JsonObject object) {
                         String icsContent = object.getString("ics");
-                        eventService.importIcal(calendarId, icsContent, user, defaultResponseHandler(request));
+                        JsonObject requestInfo = new JsonObject();
```

Comment:
put info necessary to the i18n translation in a Jsonobject and pass it as an argument to the method